content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -270,6 +270,7 @@ kubectl get pod mypod -o yaml | sed 's/\(image: myimage\):.*$/\1:v4/' | kubectl
 
 kubectl label pods my-pod new-label=awesome                      # Add a Label
 kubectl label pods my-pod new-label-                             # Remove a label
+kubectl label pods my-pod new-label=new-value --overwrite        # Overwrite an existing value
```

Comment:
```suggestion
kubectl label pods my-pod new-label=new-value --overwrite        # Overwrite an existing value
kubectl label pods my-pod new-label-                             # Remove a label
```

Reply:
@Tariqs Do you agree with the suggested change?

Reply:
@Tariqs , what do you think of Tim's suggestion?